### PR TITLE
CocoaPods Support

### PR DESCRIPTION
**Pull request by Sadmansamee**
Added support for cocoapods

### CODE REVIEW

File: SwiftyDflts.podspec [hunk 1]
```diff
@@ -0,0 +1,23 @@
...
+
+DESC
+
+s.homepage     = "https://github.com/sindresorhus/Defaults"
+s.license = { :type => "MIT", :file => "license" }
+s.author = { "sindresorhus" => "sindresorhus@gmail.com" }
+s.social_media_url = "https://sindresorhus.com"
+s.swift_version = "4.0"
+s.ios.deployment_target = "9.0"
+s.source = { :git => "https://github.com/sindresorhus/Defaults.git", :tag => s.version }
```
[the remark this answers is not in the document]
Comment (Sadmansamee) [reply SwiftyDflts.podspec]: SomeHow tag with ```v``` doesn't work, it needs to have like this ```0.2.2```

File: SwiftyDflts.podspec [hunk 2]
```diff
@@ -0,0 +1,23 @@
...
+Codable support: You can store any Codable value, like an enum.
+Debuggable: The data is stored as JSON-serialized values.
+Lightweight: It's only ~100 lines of code.
+
+DESC
+
+s.homepage     = "https://github.com/sindresorhus/Defaults"
+s.license = { :type => "MIT", :file => "license" }
+s.author = { "sindresorhus" => "sindresorhus@gmail.com" }
+s.social_media_url = "https://sindresorhus.com"
```
[the remark this answers is not in the document]
Comment (Sadmansamee) [reply SwiftyDflts.podspec]: I will update

File: SwiftyDflts.podspec [hunk 3]
```diff
@@ -0,0 +1,23 @@
+Pod::Spec.new do |s|
+s.name = "SwiftyDflts"
```
[the remark this answers is not in the document]
Comment (Sadmansamee) [reply SwiftyDflts.podspec]: With that module cocoapods isn't available, all of them are used as ```Defaults``` or ```SwiftDefaults```, ```SwiftyDefaults```

[the remark this answers is not in the document]
Comment (Sadmansamee) [reply SwiftyDflts.podspec]: Do you have any suggestion regarding this?

File: SwiftyDflts.podspec [hunk 4]
```diff
@@ -0,0 +1,23 @@
...
+Debuggable: The data is stored as JSON-serialized values.
+Lightweight: It's only ~100 lines of code.
+
+DESC
+
+s.homepage     = "https://github.com/sindresorhus/Defaults"
+s.license = { :type => "MIT", :file => "license" }
+s.author = { "sindresorhus" => "sindresorhus@gmail.com" }
+s.social_media_url = "https://sindresorhus.com"
+s.swift_version = "4.0"
```
[the remark this answers is not in the document]
Comment (Sadmansamee) [reply SwiftyDflts.podspec]: Updated

File: SwiftyDflts.podspec [hunk 5]
```diff
@@ -0,0 +1,23 @@
...
+s.ios.deployment_target = "9.0"
```
[the remark this answers is not in the document]
Comment (Sadmansamee) [reply SwiftyDflts.podspec]: Updated

File: SwiftyDflts.podspec [hunk 6]
```diff
@@ -0,0 +1,23 @@
...
+s.source = { :git => "https://github.com/sindresorhus/Defaults.git", :tag => s.version }
+s.source_files =["Sources/*.swift", "Sources/Defaults.swift"]
```
[the remark this answers is not in the document]
Comment (Sadmansamee) [reply SwiftyDflts.podspec]: Updated

File: SwiftyDflts.podspec [hunk 7]
```diff
@@ -0,0 +1,23 @@
...
+Strongly typed: You declare the type and default value upfront.
+Codable support: You can store any Codable value, like an enum.
+Debuggable: The data is stored as JSON-serialized values.
+Lightweight: It's only ~100 lines of code.
+
+DESC
+
+s.homepage     = "https://github.com/sindresorhus/Defaults"
+s.license = { :type => "MIT", :file => "license" }
+s.author = { "sindresorhus" => "sindresorhus@gmail.com" }
```
[the remark this answers is not in the document]
Comment (Sadmansamee) [reply SwiftyDflts.podspec]: Updated

File: SwiftyDflts.podspec (hunk 2 above)
[the remark this answers is not in the document]
Comment (Sadmansamee) [reply SwiftyDflts.podspec]: Updated

File: SwiftyDflts.podspec [hunk 8]
```diff
@@ -0,0 +1,23 @@
...
+s.social_media_url = "https://sindresorhus.com"
+s.swift_version = "4.0"
+s.ios.deployment_target = "9.0"
+s.source = { :git => "https://github.com/sindresorhus/Defaults.git", :tag => s.version }
+s.source_files =["Sources/*.swift", "Sources/Defaults.swift"]
+s.requires_arc = true
```
[the remark this answers is not in the document]
Comment (Sadmansamee) [reply SwiftyDflts.podspec]: Updated

File: SwiftyDflts.podspec (hunk 3 above)
[the remark this answers is not in the document]
Comment (sindresorhus) [reply SwiftyDflts.podspec]: You can use `Defaults`. I'm getting the name.

File: SwiftyDflts.podspec (hunk 1 above)
[the remark this answers is not in the document]
Comment (sindresorhus) [reply SwiftyDflts.podspec]: I'm saying, you need to fix this here. I'm not going to change how I do tags.

**Comment by sindresorhus** [comment]
I appreciate the PR, but there are just too many incorrect things in here that it's easier to just do it myself :)

**Comment by sindresorhus** [comment]
Now available on CocoaPods: https://cocoapods.org/pods/Defaults

**Pull request closed without merging**